feedback_prompt_learning/data/task/bigbench.py

```
@@ -0,0 +1,204 @@
+        
+        for example in original_examples:
+            question = example['input']
+            if 'task_prefix' in data.keys():
+                task_prefix = data['task_prefix'].strip()
+                question = task_prefix + "\n" + question
+            
+            target_scores = example['target_scores']
+            
+            # Generating options and answer
+            options = list(target_scores.keys())
+            answer = [chr(65 + i) for i, option in enumerate(options) if target_scores[option] == 1][0]
```

Ruff (0.14.7)

Prefer `next(...)` over single element slice

Replace with `next(...)`

(RUF015)